bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -139,6 +139,7 @@ BCStateTran::BCStateTran(const Config &config, IAppState *const stateApi, DataSt
       maxNumOfStoredCheckpoints_{0},
       numberOfReservedPages_{0},
       cycleCounter_(0),
+      handoff_{std::make_unique<concord::util::Handoff>(config_.myReplicaId)},
```

Comment:
nit:
Do we need to specify concord::util explicitly?
I see its already specified around line 50.

Same for chrono on line around 130 in BCStateTrans.hpp